src/components/checkbox/checkbox.scss

```diff
@@ -4,14 +4,15 @@
 
 :host {
     --mdc-checkbox-checked-color: var(--mdc-theme-primary);
+    --width-of-checkbox: 2.5rem;
```

Comment:
Is this an internal variable, or something we will document and let the consumer set? Because if it's the latter, it would be great if it had a more easily understandable name than `--constructed-size-of-checkbox` 😅

If it's internal, then the reader of the code can be expected to know about MDC, and `--constructed-size-of-checkbox` may well be a better name than `--width-of-checkbox` 👍

Reply:
Yes it's just internal. But since we're trying to explain to  our colleagues and contributors: "why this code?", I suggested the above :) 

We could also turn it to a SASS variabel instead `$constructed-size-of-checkbox` so that nobody thinks it's an externally manipulatable value. What do you think?